### PR TITLE
Devws for shina system containin 66

### PR DESCRIPTION
**Pull request by shinasys**

### CODE REVIEW

**Comment by shinasys** [comment]
@greens https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/63302#issuecomment-840832581 

> I imagine that difficulty you were having was because you had runLocally and executeCommandsLocally set to true.

So I deleted the runLocally and executeCommandLocally parameters and I added contact sensor again. 
Please checking.

**Comment by greens** [comment]
@shinasys did that solve your problem? I don't have your device.

**Comment by shinasys** [comment]
@greens Yes. I tested it.

File: devicetypes/shinasys/sihas-multipurpose-sensor.src/sihas-multipurpose-sensor.groovy [hunk 1]
```diff
@@ -121,7 +123,11 @@ private Map parseIasMessage(String description) {
 
 private Map translateZoneStatus(ZoneStatus zs) {
     // Some sensor models that use this DTH use alarm1 and some use alarm2 to signify motion
-    return (zs.isAlarm1Set() || zs.isAlarm2Set()) ? getMotionResult('active') : getMotionResult('inactive')
+    if(isDSM300()) {
```
Comment (PKacprowiczS) [inline devicetypes/shinasys/sihas-multipurpose-sensor.src/sihas-multipurpose-sensor.groovy]: nit: spaces after `if`.

File: devicetypes/shinasys/sihas-multipurpose-sensor.src/sihas-multipurpose-sensor.groovy [hunk 2]
```diff
@@ -199,6 +217,12 @@ def refresh() {
         refreshCmds +=  zigbee.enrollResponse()
     }
 
+    if(isDSM300()) {
+    	refreshCmds += zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, POWER_CONFIGURATION_BATTERY_VOLTAGE_ATTRIBUTE)
+    	refreshCmds += zigbee.readAttribute(zigbee.IAS_ZONE_CLUSTER, zigbee.ATTRIBUTE_IAS_ZONE_STATUS)        
+    	refreshCmds +=  zigbee.enrollResponse()
```
Comment (PKacprowiczS) [inline devicetypes/shinasys/sihas-multipurpose-sensor.src/sihas-multipurpose-sensor.groovy]: nit: one space too many after `+=`

**Review by PKacprowiczS — CHANGES_REQUESTED** [review]
Overall, looks good. I'd squash those commits if I were you.

**Review by greens — APPROVED** [review]

**Pull request merged**